exchanges/exchange.go

```diff
@@ -793,7 +793,7 @@ type ModifyOrder struct {
 	Amount          float64
 	LimitPriceUpper float64
 	LimitPriceLower float64
-	Currency        pair.CurrencyPair
+	Currency        currency.Pair
```

Comment:
Is Pair or CurrencyPair better name for Currency, and Currency for me points more to a single "Code" then a "Pair" of Codes. CurrencyPair harmonizes things with OrderCancellation struct